modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ImportTaskResourcePerformanceTest.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.batch.engine.resource.v1_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.headless.batch.engine.client.dto.v1_0.ImportTask;
+import com.liferay.headless.batch.engine.client.http.HttpInvoker;
+import com.liferay.headless.batch.engine.client.serdes.v1_0.ImportTaskSerDes;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.test.randomizerbumpers.UniqueStringRandomizerBumper;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.LinkedHashMapBuilder;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.io.Closeable;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Vendel Toreki
+ */
+@RunWith(Arquillian.class)
+public class ImportTaskResourcePerformanceTest
+	extends BaseTaskResourcePerformanceTestCase {
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		BaseTaskResourcePerformanceTestCase.setUpClass();
+
+		_jsonTemplates = LinkedHashMapBuilder.put(
+			"com.liferay.headless.admin.user.dto.v1_0.UserAccount",
+			_createUserAccountJSONTemplate()
+		).build();
+	}
+
+	@Test
+	public void testPostImportTaskWithUserAccount() throws Exception {
+		_testPostImportTask(
+			"com.liferay.headless.admin.user.dto.v1_0.UserAccount");
+	}
+
+	private static String _createUserAccountJSONTemplate() throws Exception {
+		return JSONUtil.put(
+			"additionalName", ""
+		).put(
+			"alternateName", "[$ALTERNATE_NAME$]"
+		).put(
+			"birthDate", "1977-01-01T00:00:00Z"
+		).put(
+			"customFields", JSONFactoryUtil.createJSONArray()
+		).put(
+			"dashboardURL", ""
+		).put(
+			"dateCreated", "2021-05-19T16:04:46Z"
+		).put(
+			"dateModified", "2021-05-19T16:04:46Z"
+		).put(
+			"emailAddress", "[$EMAIL_ADDRESS$]"
+		).put(
+			"familyName", "[$FAMILY_NAME$]"
+		).put(
+			"givenName", "[$GIVEN_NAME$]"
+		).put(
+			"jobTitle", ""
+		).put(
+			"keywords", JSONFactoryUtil.createJSONArray()
+		).put(
+			"name", "[$GIVEN_NAME$] [$FAMILY_NAME$]"
+		).put(
+			"organizationBriefs", JSONFactoryUtil.createJSONArray()
+		).put(
+			"profileURL", ""
+		).put(
+			"roleBriefs",
+			JSONUtil.put(
+				JSONUtil.put(
+					"id", 20113
+				).put(
+					"name", "User"
+				))
+		).put(
+			"siteBriefs",
+			JSONUtil.put(
+				JSONUtil.merge(
+					JSONUtil.put(
+						"id", 20127
+					).put(
+						"name", "Global"
+					),
+					JSONUtil.put(
+						"id", 20125
```

Comment:
This info is hardcoded. Is it always the same identifier?

Reply:
I agree with you. I think we should consider the default case to simplify the JSON generation, at least for this first approach